scripts/gen_type_asserts.py

```diff
@@ -0,0 +1,70 @@
+import collections.abc as c
+from itertools import islice
+from pathlib import Path
+from sys import argv
+from textwrap import dedent
+
+
+def get_funcs(models: c.Iterable[Path]) -> c.Iterator[str]:
+    for path in models:
+        with path.open() as f:
+            buf = []
+            in_func = False
+            # skip first 14 lines of boilerplate
+            for raw in islice(f.readlines(), 14, None):
+                line = raw.rstrip()
+                buf.append(line)
+                if line.startswith("func"):
+                    in_func = True
+                elif line.startswith("}") and in_func:
+                    in_func = False
+                    yield "\n".join(buf)
+                    buf.clear()
```

Comment:
I'd take the approach of re-ordering in the file by keeping the header, as shown here: https://github.com/kubeflow/model-registry/pull/404/files#diff-30505a65ba1ad63e91b0b363704b01d852f43625afea32db6355c7626679a7aaR8-R28

this way we don't have to rely on a fixed `14` my2c